.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: go
+
+go:
+    - 1.7
+    - 1.8
+    - tip
+
+matrix:
+    allow_failures:
```

Comment:
Let's use fast finish too:
https://docs.travis-ci.com/user/customizing-the-build#Fast-Finishing